core/wiki/macros/list-links-horizontal-draggable.tid

```diff
@@ -0,0 +1,68 @@
+title: $:/core/macros/list-links-horizontal-draggable
+tags: $:/tags/Macro $:/tags/Global
+
+\whitespace trim
+\procedure list-links-horizontal-draggable-caption()
+<$let tv-wikilinks="no">
+	<span class="tc-small-gap-right">
+		<$transclude field="caption">
+			<$view field="title"/>
+		</$transclude>
+	</span>
+</$let>
+\end
+
+\procedure list-links-horizontal-draggable(tiddler,field:"list",emptyMessage,type:"span",subtype:"span",class:"",itemTemplate,dragHandle:"⠿",enable:"yes")
```

Comment:
It is not appropriate to use Unicode characters as UI elements. Unicode characters have semantic meaning, and here those semantics are inappropriate.

I think we need a new `$:/core/images/drag-handle` SVG image.